scrap_zenodo.py

```diff
@@ -3,11 +3,12 @@
 
 import requests
 import dotenv
+import pandas as pd
 
 
 def read_zenodo_token():
     """Read file Zenodo token from disk."""
-    dotenv.load_dotenv(".env")
+    dotenv.load_dotenv(".env.txt")
```

Comment:
Could you please keep the filename `.env` for the Zenodo API key? This is the standard filename for the dotenv package.